<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtsAlarmReport</class>
 <widget class="QWidget" name="AtsAlarmReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="devicecomboBox">
        <item>
         <property name="text">
          <string>所有设备</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>应用服务器</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>接口服务器</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>通讯服务器</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数据库项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中央控制工作站</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>现地工作站</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>维护工作站</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设备类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="typecomboBox">
        <item>
         <property name="text">
          <string>不限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>板级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>外围</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其它</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>故障类型</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="faultcomboBox">
        <item>
         <property name="text">
          <string>不限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>硬件故障</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>软件故障</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>报警级别</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="faulttypecomboBox">
        <item>
         <property name="text">
          <string>不限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>一级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>二级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三级</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>到</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="findbutton">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QPushButton" name="exportbutton">
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>143</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="firstbutton">
     <property name="text">
      <string>首页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="prebutton">
     <property name="text">
      <string>上一页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="page">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="nextbutton">
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="lastbutton">
     <property name="text">
      <string>尾页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="totalcount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
